site/content/integrate/plugins/server/reference.md

```diff
@@ -4,6 +4,7 @@ heading: "Mattermost Server Reference"
 description: "The plugin package is used by Mattermost server plugins written in Go and enables you to manage and interact with the plugin environment."
 date: 2018-07-10T00:00:00-05:00
 weight: 10
+aliases: [/extend/plugins/server/reference/]
```

Comment:
Is this used to support existing links to the `extend` subpath?

Reply:
@mickmister If I understand your answer correctly, yes. Aliases in Hugo create automatic redirects for the file path to send traffic to this page. I tested these locally and they perform as expected.